library/src/morpheus-master/sass/base/_defaults.scss

```diff
@@ -215,6 +215,20 @@ button[class="disabled"]:hover,
 	text-decoration: none;
 }
 
+input[type="checkbox"].formButtonDisabled{
+	@include border-radius(3px);
+	&:checked{
+		background-image: url("images/check.png") !important;
```

Comment:
Yeah, if there's a better way to do it, I'm all ears. Or if the community doesn't want this and nobody has the time to resolve it in a different way, I can close this PR.

This is just how we did it locally without spending a lot of time on the issue. For us, the ugliness of the issue needed to be resolved without taking away too much time from other high priority tasks that required a lot of time and development.